R/options.R

```diff
@@ -179,6 +175,7 @@ polars_options = function() {
     df_knitr_print = getOption("polars.df_knitr_print"),
     do_not_repeat_call = getOption("polars.do_not_repeat_call"),
     int64_conversion = getOption("polars.int64_conversion"),
+    limit_max_threads = getOption("polars.limit_max_threads") %||% "Not set (NULL)",
```

Comment:
if the default is to limit the number of threads to 2, shouldn't it be `%||% TRUE`?

Reply:
TRUE doesn't make sense. This is because there is a possibility that the behavior will be changed on the lib side. Only an explicit FALSE is meaningful

Reply:
Sorry I don't understand that. Suppose I install the package from CRAN and I don't set any option anywhere. This means that there is an automatic limit to use 2 threads. If I do `polars_options()`, what would be printed?

`limit_max_threads    TRUE` or `limit_max_threads    Not set (NULL)`?

If it's `TRUE`, I think it's fine, but if it's `Not set (NULL)` then that's confusing because I would understand it as "there's no limit in the number of threads", while actually there is one.

Reply:
The default is changed to `!polars_info()$features$disable_limit_max_threads`